app/controllers/standup_meetings_controller.rb

```diff
@@ -0,0 +1,14 @@
+class StandupMeetingsController < ApplicationController
+  before_action :set_standup_meeting_group
+
+  def edit
+    @standup_meeting = StandupMeeting.find(params[:id])
```

Comment:
should just have an `includes` instead of the `before_action`. (and you can set `@standup_meeting_group` in here if needed.